.github/workflows/tests.yml

```diff
@@ -64,43 +64,43 @@ jobs:
 
             if (label_names.includes('CI-syntax-only')) {
               console.log('CI-syntax-only label found. Skipping tests.')
-              console.log("::set-output name=run-tests::false");
+              console.log('::set-output name=syntax-only::true');
             }
             else {
```

Comment:
This is unrelated but is there a compelling reason why the `else` is on a separate line instead of the usual `} else {` format?:

```javascript
if (something) {
} else {
}
```

If not, I'll tidy this up while I'm at it. I know there are folks who will argue that you can easily comment out the `else` block with the `}\nelse {` format but I'm decidedly not in this camp and I feel like `} else {` is the more common style in JavaScript.

Reply:
I don't think we have a preferred style for Homebrew because we rarely use JavaScript for anything. I'd say do what you think is most readable; I'm fine either way (and I don't think we need to be overly concerned with commenting-out because we don't commit commented-out code most of the time).